min/lib/Minify/Controller/MinApp.php

```diff
@@ -136,13 +136,14 @@ public function setupSources($options) {
                 $base = '/';
             }
             $allowDirs = array();
+            $document_root = realpath($_SERVER['DOCUMENT_ROOT']);
```

Comment:
My [latest commit](https://github.com/mrclay/minify/commit/de39966e9c2ba90b7e400f92b867f6592e34046f) handles this for the upcoming 3.0. In 2.x, you should manually set `$_SERVER['DOCUMENT_ROOT']`.